### DISCUSSION TITLE
Add config area for which idle notifications to be alerted for

### DISCUSSION THREAD

**Discussion by Jalopyy** [runelite/runelite#15041] [Ideas]
Allow a config area so we can decide which activities we want to be alerted to. If I'm not making potions en masse, I don't care about being a bit inefficient with my mixing. Same can be said for things like only making a single inventory of things, or only a few at a time. This way people can turn modules on/off depending on what skills they're training at the time.

**Comment by deathbeam** [comment]
Best practice for everyone either looking at issues or solving them is to split multiple stuff to multiple issues. Like, 4. is duplicate of another issue. 3. is probably your misconfiguration somewhere in system. 2. is due to Jagex reusing animations, I don't think we can do much about it.

**Comment by Jalopyy** [comment]
I'll delete 2-4 then. And I'll not make multi-faceted tickets in the future, then.
